src/main/java/karuna/karuna_backend/security/config/WebSecurityConfig.java

```diff
@@ -42,11 +43,18 @@ SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
                                 "/swagger-ui/**",
                                 "/swagger-ui.html",
                                 "/v3/api-docs/**",
-                                "/swagger-resources/**",
-                                "/visitors/messages/**")
+                                "/swagger-resources/**")
                         .permitAll()
+                        .requestMatchers(HttpMethod.POST, "/visitors/messages").hasAnyRole("USER", "ADMIN")
```

Comment:
Is not needed .anyRequest().authenticated() make the same thing, i write u on dc

Reply:
we dont have more roles than user and admin so authenticated is equal to be user or admin